src/Transaction.js

```diff
@@ -521,7 +521,7 @@ class Transaction {
     async _spendUTXO(accountUTXO, account, currentValue, changeAddress, changeAddressCurve) {
         try {
             accountUTXO = accountUTXO["ValueStores"]
-            while (true) {
+            while (Constants.ReqCondition) {
```

Comment:
`constant.ReqCondition` here addresses the rule https://eslint.org/docs/rules/no-constant-condition

Reply:
Same as above